examples/step-47/step-47.cc

```diff
@@ -837,23 +837,27 @@ namespace Step47
       std::vector<SymmetricTensor<2, dim>> exact_hessians(n_q_points);
       std::vector<Tensor<2, dim>>          hessians(n_q_points);
       for (auto &cell : dof_handler.active_cell_iterators())
-        {
-          fe_values.reinit(cell);
-          fe_values[scalar].get_function_hessians(solution, hessians);
-          exact_solution.hessian_list(fe_values.get_quadrature_points(),
-                                      exact_hessians);
-
-          double local_error = 0;
-          for (unsigned int q_point = 0; q_point < n_q_points; ++q_point)
-            {
-              local_error +=
-                ((exact_hessians[q_point] - hessians[q_point]).norm_square() *
-                 fe_values.JxW(q_point));
-            }
-          error_per_cell[cell->active_cell_index()] = std::sqrt(local_error);
-        }
+        if (cell->is_locally_owned())
```

Comment:
The code is sequential, why do you introduce this check?

Reply:
I can remove this check. But, I think it does not harm here: on the one hand these lines of code mimic in a certain sense `VT::do_integrate_difference()` (the result is not accumulated direct into a temporal variable but into a vector, which is accessed by `cell->active_cell_index()`) and on the other hand this one less change one has to perform if that person wants to parallelize the code (which already happened at least 2 times of which I am aware of).

Reply:
I don't feel strongly about it, but I think we avoid changes like this.
Anybody else have an opinion here?

Reply:
I have removed it.

Reply:
I was going to suggest that we follow @tjhei's opinion here. I'm glad that it's now a non-issue.